requirements.txt

```diff
@@ -7,3 +7,6 @@ pytz==2019.3
 requests==2.23.0
 six==1.14.0
 urllib3==1.25.9
+pandas==0.25.0
+sklearn==0.21.3
```

Comment:
Did you mean scikit-learn?

Reply:
Ah yes! My bad, it should be `scikit-learn` instead of `sklearn`.